src/Symfony/Bridge/Twig/Form/TwigRenderer.php

```diff
@@ -17,25 +17,10 @@
 /**
  * @author Bernhard Schussek <bschussek@gmail.com>
  */
-class TwigRenderer extends FormRenderer implements TwigRendererInterface
+class TwigRenderer extends FormRenderer
```

Comment:
This class has no value anymore. `Symfony\Component\Form\FormRenderer` can be used directly now. So TwigRenderer  should be deprecated and removed IMO.